<?xml version="1.0"?>
<launch>

    <arg name="start_pt" default= '-9.16926193237, -15.6781167984, 0.8978652, 0'/>
    <arg name="use_rviz" default="true"/>
    <arg name="use_matplotlib" default="false"/>
    <arg name="map_file" default="$(find svea)/src/scripts/maps/floor2.simplemap" />

    <group if="$(eval use_rviz == true)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find svea)/src/scripts/rviz/SVEA_indoor_team3.rviz" />
    </group>

    <node pkg="mrpt_map" type="map_server_node" name="map_server_node" output="screen">
        <param name="debug" value="false"/>
        <param name="frame_id" value="map"/>
        <param name="frequency" value="1"/>
        <param name="ini_file" value="$(find svea_sensors)/params/mrpt/pf_localization.ini"/>
        <param name="map_file" value="$(arg map_file)"/>
    </node>

    <node pkg="svea" type="floor2_example.py" name="floor2_example" output="screen">
        <param name="start_pt" value="$(arg start_pt)"/>
        <param name="use_rviz" value="$(arg use_rviz)"/>
        <param name="use_matplotlib" value="$(arg use_matplotlib)"/>
    </node>

</launch>
